Assets/Scripts/Models/Furniture.cs

```diff
@@ -521,41 +538,41 @@ public void ReadXmlPrototype(XmlReader reader_parent)
                     Job.JobPriority.High );
 
                 World.current.SetFurnitureJobPrototype(j, this);
-
                 break;
+            
             case "OnUpdate":
-
                 string functionName = reader.GetAttribute("FunctionName");
                 RegisterUpdateAction(functionName);
-                    break;
+                break;
+            
             case "OnInstall":
                 // Called when obj is installed
                 string functionInstallName = reader.GetAttribute("FunctionName");
                 RegisterInstallAction(functionInstallName);
-
                 break;
+            
```

Comment:
Could you please clean up trailing whitespace? Not useful and makes for uglier diff/merge. :)